pontoon/base/models.py

```diff
@@ -192,6 +204,69 @@ def latest_activity(self, project=None):
         return translations.latest_activity()
 
 
+@receiver(pre_save, sender=Locale)
+def create_locale_translators_group(sender, instance,  raw, **kwargs):
+    """
+    Creates respective translators group in django's permission system.
+    Managers should be able to translate and manage locale.
+    """
+    if raw:
```

Comment:
Why not abort if the locale isn't being created as well? I don't think we'd want to do this creation check on _every_ save.